sentry/resource_sentry_project_rule_test.go

```diff
@@ -57,28 +56,28 @@ func TestAccSentryRule_basic(t *testing.T) {
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckSentryRuleExists("sentry_rule.test_rule", &rule, testOrganization, projectSlug),
 					testAccCheckSentryRuleAttributes(&rule, &testAccSentryRuleExpectedAttributes{
-						Name: "Important Issue",
+						Name:        "Important Issue",
 						ActionMatch: "all",
-						Frequency: 1440,
+						Frequency:   1440,
 						Environment: "prod",
 						Actions: []sentryclient.RuleAction{
 							{
-								ID: "sentry.rules.actions.notify_event.NotifyEventAction",
-								Name: "Send a notification (for all legacy integrations)",  // Default name added by Sentry
+								ID:   "sentry.rules.actions.notify_event.NotifyEventAction",
+								Name: "Send a notification (for all legacy integrations)", // Default name added by Sentry
 							},
 						},
 						Conditions: []sentryclient.RuleCondition{
 							{
-								ID: "sentry.rules.conditions.event_frequency.EventFrequencyCondition",
-								Value: 100,
-								Name: "An issue is seen more than 100 times in 1m",
+								ID:       "sentry.rules.conditions.event_frequency.EventFrequencyCondition",
+								Value:    100,
+								Name:     "The issue is seen more than 100 times in 1m",
 								Interval: "1m",
 							},
 							{
-								ID: "sentry.rules.conditions.event_frequency.EventUniqueUserFrequencyCondition",
+								ID:       "sentry.rules.conditions.event_frequency.EventUniqueUserFrequencyCondition",
 								Interval: "1m",
-								Name: "An issue is seen by more than 25 users in 1m",
-								Value: 25,
+								Name:     "The issue is seen by more than 25 users in 1m",
```

Comment:
Otherwise I got
```
=== RUN   TestAccSentryRule_basic
--- FAIL: TestAccSentryRule_basic (1.61s)
    testing.go:654: Step 0 error: Check failed: Check 2/2 error: got conditions: [{ID:sentry.rules.conditions.event_frequency.EventFrequencyCondition Name:The issue is seen more than 100 times in 1m Attribute: Match: Value:100 Key: Interval:1m} {ID:sentry.rules.conditions.event_frequency.EventUniqueUserFrequencyCondition Name:The issue is seen by more than 25 users in 1m Attribute: Match: Value:25 Key: Interval:1m}]
        ; want [{ID:sentry.rules.conditions.event_frequency.EventFrequencyCondition Name:An issue is seen more than 100 times in 1m Attribute: Match: Value:100 Key: Interval:1m} {ID:sentry.rules.conditions.event_frequency.EventUniqueUserFrequencyCondition Name:An issue is seen by more than 25 users in 1m Attribute: Match: Value:25 Key: Interval:1m}]
```